.deployment.config.json

```diff
@@ -9,9 +10,12 @@
       "sequential": ["dev", "qa", "prd"]
     },
     "team_jenkins": "searchuibuilds",
-    "start_environment_automatically": false,
+    "start_environment_automatically": true,
```

Comment:
We don't want `dev` and `qa` to require manual approval in order to start their phases.

`qa` is now started automatically when there's a release bump commit.